src/middlewared/middlewared/plugins/failover_/event_linux.py

```diff
@@ -661,6 +663,11 @@ def vrrp_backup(self, job, fobj, ifname, event):
         logger.info('Stopping rrdcached')
         self.run_call('service.stop', 'rrdcached', self.HA_PROPAGATE)
 
+        for vm in (self.run_call('vm.query', [['status.state', '=', 'RUNNING']]) or []):
+            self.run_call('vm.poweroff', vm['id'], True)
```

Comment:
`poweroff` only accepts vm id, can you please clarify usage of `True` ?

Reply:
@themylogin  ping

Reply:
Sorry, bad automatic backport. On the second though... @yocalebo do you recall why there was no `vm.poweroff` in SCALE HA code?

Reply:
@themylogin because the operation is serial and each VM has a timeout before we forcefully turn them off. So this line could add 10's of minutes to the failover process. More importantly, however, by the time this code is run the zpool has been exported so I'm not sure what the state of the VM's would be in by the time we get here. So I'm not really sure it matters that we power them off "cleanly". Lastly, VM's aren't "officially" supported in HA environments so I'm not really sure what the approach is here.

Reply:
I'd rather remove the `vm.poweroff` logic in this PR since the original ticket is for the TrueCommand service. I'd rather open a separate ticket and discuss it there.